src/RequestHandler/outofprocess/processmanager.cxx

```diff
@@ -173,9 +173,7 @@ PROCESS_MANAGER::GetProcess(
                 // terminate existing process that is not ready
                 // before creating new one.
                 //
-
-                //todo:
-                //ShutdownProcessNoLock( m_ppServerProcessList[dwProcessIndex] );
+                ShutdownProcessNoLock( m_ppServerProcessList[dwProcessIndex] );
```

Comment:
Did you forget to uncomment this before?

Reply:
seems true